os/cluster-architectures.md

```diff
@@ -86,38 +96,42 @@ Following the guide for each of the [supported platforms](https://coreos.com/doc
 |------|-----------|-------------|------------|
 | Low | Development/Testing | Minutes | No |
 
-When you're first getting started with Container Linux, it's common to frequently tweak your cloud-config which requires booting/rebooting/destroying many machines. Instead of being slowed down and distracted by generating new discovery URLs and bootstrapping etcd, it's easier to start a single etcd node.
+When getting started with Container Linux, you may find yourself booting/rebooting/destroying many machines. Instead of slowing down and distracting yourself, generating new discovery URLs and bootstrapping etcd. Instead, start a single etcd node and build your cluster around that.
 
-You are now free to boot as many machines as you'd like as test workers that read from the etcd node. All the features of fleet, Locksmith, and etcdctl will continue to work properly, but will connect to the etcd node instead of using a local etcd instance. Since etcd isn't running on all of the machines, you'll gain a little bit of extra CPU and RAM to play with.
+You can now boot as many machines as you'd like as test workers that read from the etcd node. All the features of Locksmith and etcdctl will continue to work properly but will connect to the etcd node instead of using a local etcd instance. Since etcd isn't running on all of the machines you'll gain a little bit of extra CPU and RAM to play with.
 
-This environment is now set up to take a beating. Pull the plug on a machine and watch fleet reschedule the units, max out the CPU, etc.
+Once this environment is set up, it'll be ready to take a beating. Pull the plug on a machine and watch Kubernetes reschedule the units, max out the CPU, etc.
```

Comment:
... it's ready to be tested. Destroy a machine, and watch Kubernetes reschedule the units, max out the CPU, and rebuild your setup automatically.